concepts/booleans/about.md

```diff
@@ -1,3 +1,24 @@
 # About
 
-TODO
+Booleans in Rust are represented by the `bool` type, which values can be either `true` or `false`.
+
+Rust supports one [boolean negation operator][negation operators]: `!` (Not).
+
+Rust supports six [comparison operators][comparison operators]: `==` (Equal), `!=` (Not equal), `>`	(Greater than), `<` (Less than), `>=`
```

Comment:
```suggestion
Rust supports six [comparison operators][comparison operators]: `==` (Equal), `!=` (Not equal), `>` (Greater than), `<` (Less than), `>=`
```

Hard tab here also. I guess you copied the text from somewhere? If so, might be necessary to attribute it.